src/solc_0.6/common/BaseWithStorage/MetaTransactionReceiver.sol

```diff
@@ -16,6 +15,7 @@ contract MetaTransactionReceiver is Admin {
     /// @param enabled set whether the metaTransactionProcessor is enabled or disabled.
     function setMetaTransactionProcessor(address metaTransactionProcessor, bool enabled) public {
         require(msg.sender == _admin, "only admin can setup metaTransactionProcessors");
+        require(metaTransactionProcessor != address(0), "MetaTransactionReceiver: zero address");
```

Comment:
The auditor asked to change `_setMetaTransactionProcessor()`
not `setMetaTransactionProcessor`

Not sure what's the best since we have already a require in `setMetaTransactionProcessor`

Reply:
moved the require to _setMetaTransactionProcessor()

https://github.com/thesandboxgame/sandbox-smart-contracts/pull/799/commits/f1ec408c392a2e24b63722c243a00f5ad091390b